data.json

```diff
@@ -377,6 +377,15 @@
     "dateAdded": "2018-10-24T11:42:09.000Z",
     "tags": ["clothing", "stickers"]
   },
+  {
+    "name": "GeeksForGeeks",
+    "difficulty": "medium",
+    "description": "Solve daily coding problems and earn coins,and redeem swags",
```

Comment:
Please refine a bit the description
Add missing space and describe what are the swag.